l10n_ch_fds_upload_sepa/wizards/fds_files_import_tobankstatments_wizard.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, api
+
+import base64
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class FdsFilesImportToBankStatementsWizardPain000(models.TransientModel):
+    _inherit = 'fds.files.import.tobankstatments.wizard'
+
+    @api.multi
+    def _import2bankStatements(self, fds_files_ids):
+        account_pain000 = self.env['account.pain000.parser']
+        for fds_file in fds_files_ids:
+            for pf_file in fds_file:
+                values = {'data_file': pf_file.data}
+                bs_import_obj = self.env['account.bank.statement.import']
+                bank_wiz_imp = bs_import_obj.create(values)
```

Comment:
I think this step is not needed. Can't you read the file data only by doing `decoded_file = base64.b64decode(pf_file.data)` ? Or is the statement import wizard doing something more?